src/Components/Components/test/Routing/RouterTest.cs

```diff
@@ -181,8 +181,7 @@ internal class TestNavigationManager : NavigationManager
         {
             public TestNavigationManager() =>
                 Initialize("https://www.example.com/subdir/", "https://www.example.com/subdir/jan");
-
-            protected override void NavigateToCore(string uri, bool forceLoad) => throw new NotImplementedException();
+            protected override void NavigateToCore(string uri, NavigationOptions options) => throw new NotImplementedException();
```

Comment:
This change can and should be reverted, to show we're not relying on subclasses providing an implementation for the new overload.